packages/features/bookings/Booker/components/OverlayCalendar/OverlayCalendar.tsx

```diff
@@ -44,11 +47,13 @@ export const OverlayCalendar = ({
         hasSession={hasSession}
         onStateChange={handleSwitchStateChange}
       />
-      <OverlayCalendarContinueModal
-        open={isOpenOverlayContinueModal}
-        onClose={handleCloseContinueModal}
-        onContinue={handleClickContinue}
-      />
+      {!isPlatform && (
```

Comment:
Don't know why we have this for overlay calendar, but it looks like some terms and conditions agreement and I don't think anything different happens with it.